.gitignore

```diff
@@ -34,6 +34,7 @@ savedmodels/
 /*.please_wait
 /auto_specials*.csv
 /weighted_mean*.csv
+/agg_gap_hours_24*.csv
```

Comment:
A file generated by one of the integration tests. Even before this change it was being generated, but not gitignored.